caproto/commandline/get.py

```diff
@@ -137,13 +136,10 @@ def main():
     #        if the functionality is not desired.
     clean_format_args(args=args)
 
-    if args.no_color:
-        set_handler(color=False)
     if args.verbose:
-        logging.getLogger('caproto.ch').setLevel('DEBUG')
-        logging.getLogger('caproto.ctx').setLevel('DEBUG')
-        if args.verbose > 2:
-            logging.getLogger('caproto').setLevel('DEBUG')
+        set_handler(color=not args.no_color, level='DEBUG')
```

Comment:
This is not the same thing. Notice that the implementation in `master` does not include a debug logger on `'caproto.circ'`. Your approach here would be much more verbose than the one we have in `master`. I suggest making a couple `StreamHandlers` here, setting them to use our formatter, and add them directly to the two loggers 'caproto.ch' and 'caproto.ctx'. I think I illustrated this earlier, but perhaps my reason for suggesting it wasn't clear at the time.